<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsAtlasCompositionWidgetBase</class>
 <widget class="QWidget" name="QgsAtlasCompositionWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>501</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Atlas generation</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_8">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QToolBox" name="toolBox">
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>450</width>
        <height>470</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Atlas options</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_9">
       <item row="2" column="0">
        <widget class="QFrame" name="mAtlasFrame">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <layout class="QGridLayout" name="gridLayout_7" rowstretch="0,0,0,0,0,0,0,0,0" columnstretch="0,0,0">
            <item row="2" column="0" colspan="2">
             <widget class="QCheckBox" name="mAtlasHideCoverageCheckBox">
              <property name="toolTip">
               <string>Hide the coverage layer when generating the output</string>
              </property>
              <property name="text">
               <string>Hidden coverage layer</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Margin around coverage</string>
              </property>
             </widget>
            </item>
            <item row="7" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Output filename expression</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QSpinBox" name="mAtlasMarginSpinBox">
              <property name="suffix">
               <string> %</string>
              </property>
              <property name="maximum">
               <number>100</number>
              </property>
              <property name="value">
               <number>10</number>
              </property>
             </widget>
            </item>
            <item row="7" column="2">
             <widget class="QToolButton" name="mAtlasFilenameExpressionButton">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
            <item row="7" column="1">
             <widget class="QLineEdit" name="mAtlasFilenamePatternEdit"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Coverage layer</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QComboBox" name="mAtlasCoverageLayerComboBox">
              <property name="contextMenuPolicy">
               <enum>Qt::NoContextMenu</enum>
              </property>
             </widget>
            </item>
            <item row="5" column="0" colspan="2">
             <widget class="QCheckBox" name="mAtlasFixedScaleCheckBox">
              <property name="text">
               <string>Fixed scale</string>
              </property>
             </widget>
            </item>
            <item row="8" column="0" colspan="2">
             <widget class="QCheckBox" name="mAtlasSingleFileCheckBox">
              <property name="text">
               <string>Single file export when possible</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="mComposerMapComboBox"/>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Composer map to use</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="5" column="0">
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>76</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <widget class="QCheckBox" name="mUseAtlasCheckBox">
         <property name="text">
          <string>Generate an atlas</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
